peachjam/migrations/0040_backfill_work_languages.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.16 on 2022-12-01 11:54
+
+from django.db import migrations
+
+
+def backfill_work_languages(apps, schema_editor):
+    CoreDocument = apps.get_model("peachjam", "CoreDocument")
+    for doc in CoreDocument.objects.all():
```

Comment:
This will load all fields of all documents into memory. On large collections this is huge. Instead, use .iterator(100) to do only 100 at a time. 

We've learnt the hard way to batch it like this!